test/plugin_functional/plugins/core_plugin_deep_links/public/plugin.tsx

```diff
@@ -41,7 +41,7 @@ export class CorePluginDeepLinksPlugin
               id: 'pageB',
               title: 'DL page B',
               path: '/page-b',
-              navLinkStatus: AppNavLinkStatus.visible,
+              visibleIn: ['globalSearch', 'sideNav'],
```

Comment:
Can we also have a shared const for the default app visibility? Something like `DEFAULT_APP_VISIBILITY`?

Reply:
Done in https://github.com/elastic/kibana/pull/176383/commits/6ea6e0ff2928584b97f677a3f270d1d78f95fd4a